test-packages/e2e-tests/test/sdk-metadata.spec.ts

```diff
@@ -19,10 +18,14 @@ describe('sdk-metadata', () => {
   });
 
   it('gives instruction with working link', async () => {
-    expect(getGenerationSteps().instructions).toContain(
-      linkGenerationDocumentaion
-    );
-    checkUrlExists(linkGenerationDocumentaion);
+    expect(
```

Comment:
you could add similar tests for the openapi metadata that the generator link is ok and all other specific links work.